### DISCUSSION TITLE
posts/proxmox-backup-server/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#266] [Comments]
# posts/proxmox-backup-server/

Proxmox Backup Server is an enterprise-class client-server backup software that backs up virtual machines, containers, and physical hosts.In this step by step tutorial, we install and configure Proxmox Backup Server (PBS) and back up all of our virtual machines. We’ll start with nothing and end up with a fully functional Proxmox Backup Server with a ZFS datastore you can use to back up and restore your machines today.

https://technotim.live/posts/proxmox-backup-server/

**Comment by jim8989** [comment]
Greetings sir,
As always another fine video.
You're installing PBS as a VM and from reading I do the Proxmox forum suggests to either install it on a dedicated server by itself or on a server with PVE installed but not as a VM. Did you do the VM install to simplify the video you made or is this a long term install?  If it's permanent install what settings for the VM install did you finalize on. Before I retired I was use to ESXi but I'm new to Proxmox and in the learning stage on choosing resources for a PVE VM install

Jim

**Reply by timothystewart6** [reply]
Thank you!  I would definitely go with a physical install if it's going to be your long term backup solution.  The only way I would recommend virtual is if can you actually pass through an HBA controller to it.  This would effectively separate the machine from the drives.

**Comment by evandorkins** [comment]
Excellent guidance - I always learn something if not multiple things from your videos.  Wondering if you'd advise on this gem of a setup. . . I'm going to run (for non production clearly) PBS in a vm on PVE.  Works great.  But network traffic is the bottleneck, even with a 10G connection.   Is it possible to setup a "private / internal only" network that's just meant for the PVE vms to communicate with each other and nothing else, making the i/o speeds beyond the speed of "plaid"?!  I did this on Hyper-V with some success and wondering how to do it on PVE/PBS.  I've got the PVE side covered but PBS networking options in the UI are non-existent, and my linux skills are 'meh'.  Any tips!?  Cheers and keep the videos comin!

**Comment by mrkhachaturov** [comment]
Can you please share truenas nfs dataset permission config?

**Reply by mrkhachaturov** [reply]
https://docs.benjamin-altpeter.de/network/proxmox-backup-truenas/#:~:text=Setting%20up%20NFS%20share%20in%20TrueNAS%20In%20the,Unix%20Shares%20%28NFS%29%2C%20create%20a%20new%20NFS%20share.

**Comment by lolzob** [comment]
Can you help me with a solution for this?

https://www.reddit.com/r/homelab/comments/1fdfw2x/home_server_with_limited_budget/?rdt=38203
